radicle-cli/src/commands/issue.rs

```diff
@@ -68,12 +70,14 @@ pub enum Operation {
         id: IssueId,
         reaction: Reaction,
     },
-    List,
+    List {
+        assignee: Option<cob::ActorId>,
+    },
 }
 
 #[derive(Debug)]
 pub struct Options {
-    pub op: Operation,
+    pub op: Option<Operation>,
```

Comment:
I find this change complicates a lot of the logic. I would keep it simple and default to showing all issues, which I think is a sensible default (ie. assignee: `None`)